compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -1854,6 +1854,9 @@ public BLangNode transform(CheckExpressionNode checkExpressionNode) {
     @Override
     public BLangNode transform(TypeTestExpressionNode typeTestExpressionNode) {
         BLangTypeTestExpr typeTestExpr = (BLangTypeTestExpr) TreeBuilder.createTypeTestExpressionNode();
+        if (typeTestExpressionNode.isKeyword().kind() == SyntaxKind.NOT_IS_KEYWORD) {
+            typeTestExpr.isNotIsExpr = true;
```

Comment:
How about `typeTestExpr.isNegation` insted of `typeTestExpr.isNotIsExpr`?

Reply:
Yes it makes more sense. Will change that